selenium-system-test/src/test/resources/features/common/Verify header and footer links.feature

```diff
@@ -1,6 +1,30 @@
 Feature: Verifying header and footer links on every page
 
-  @HTS-668
+  @HTS-668 @VerifyHandF
   Scenario: when users navigates through the HTS pages they see feedback, get-help and privacy links are working
     When the user has logged in and passed IV
     Then they go through the happy path they can see and access all header and footer links
+
+  @VerifyHandF
+  Scenario: checking the Header and Footer links of the pages that have not been tested so far when logged in
```

Comment:
@Jackie1988 Is there a more specific scenario name we can give for these pages?

Reply:
yeah sure